Source/WebCore/css/html.css

```diff
@@ -108,6 +108,10 @@ hr {
     border-width: 1px;
 }
 
+abbr[title], acronym[title] {
+    text-decoration: dotted underline;
+}
+
```

Comment:
This looks good, but we probably need a layout test unless it is already covered by WPT.

Reply:
I will bring test from Chromium / Blink patch because I checked WPT, we don't have any. :-)

Reply:
Done!!